### PR TITLE
V3 companies

### PR DESCRIPTION
**Pull request by sandrineay**
WIP - not ready for review!

### CODE REVIEW

File: spec/models/company_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (ssaunier) [inline spec/models/company_spec.rb]: Remove this test boilerplate file 🙏

**Comment by ssaunier** [comment]
The branch was in a weird state, I just updated it.

Could you continue working on it and use the newest CSS by Cedric for your Companies CRUD 🙏  ?

Thanks.

File: spec/models/company_spec.rb (hunk 1 above)
Comment (sandrineay) [reply spec/models/company_spec.rb]: done!

File: app/views/admin/companies/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<div id="admin-container">
+  <div class="admin-header">
+    <ul class="admin-header-breadcrumbs">
+      <li><%= link_to "Companies", admin_companies_path %></li>
+      <li><%= link_to "New Company", "#", class: "current" %></li>
```
Comment (ssaunier) [inline app/views/admin/companies/_form.html.erb]: Is it relevant for the `edit` page?

File: app/controllers/admin/companies_controller.rb [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+    def new
+      @company = Company.new
+    end
+
+    def create
+      @company = Company.new(company_params)
+      @company.save
+      if @company.save
+        redirect_to admin_company_path(@company.name)
```
Comment (ssaunier) [inline app/controllers/admin/companies_controller.rb]: Show page being the same as the edit confuses me. I think we should drop the `show` page here and have the creation (or update) redirect to the company index with a flash notification (to be integrated to the admin layout).

This UX remark comes from following a simple "Add a company" flow.

File: app/views/admin/companies/_form.html.erb (hunk 2 above)
Comment (sandrineay) [reply app/views/admin/companies/_form.html.erb]: For the ```edit``` it indeed makes sense to only have a redirect towards the index (so deleting line 5)

File: config/locales/fr.yml [hunk 4]
```diff
@@ -1,4 +1,20 @@
 fr:
+  views:
```
Comment (ssaunier) [inline config/locales/fr.yml]: FYI we won't localize the `/admin`, we'll just do English (like Kitt)

**Pull request merged**